README.md

```diff
@@ -34,6 +37,16 @@ smtp:
     - OPENDKIM_DOMAINS=smtp.domain.tld
 ```
 
+### Known issues
+
+#### I see `key data is not secure: /etc/opendkim/keys can be read or written by other users` error messages.
+
+Some Docker distributions like Docker for Windows and RancherOS seems to handle
+volume permission in way that does not work with OpenDKIM default behavior of
+ensuing safe permissions on private keys.
+
+A workaround is to disable the check using a `OPENDKIM_RequireSafeKeys=no` environment variable.
```

Comment:
It may be worth noting here (at whatever level of detail you deem appropriate) that this workaround is strongly discouraged from a security perspective.

Reply:
Maybe should be even clear or is "... does not work with OpenDKIM default behavior of
ensuing safe permissions on private keys." enough?